src/functions/utils.php

```diff
@@ -325,4 +325,21 @@ function tribe_retrieve_object_by_hook( $class_name, $hook, $priority ) {
 
 		return false;
 	}
+}
+
+if ( ! function_exists( 'tribe_is_wpml_active' ) ) {
+	/**
+	 * A unified way of checking if WPML is activated.
+	 *
+	 * @since TBD
+	 *
+	 * @return boolean
+	 */
+	function tribe_is_wpml_active() {
+		if ( class_exists( 'SitePress' ) && defined( 'ICL_PLUGIN_PATH' ) ) {
```

Comment:
This function could be shortened to:
`return class_exists( 'SitePress' ) && defined( 'ICL_PLUGIN_PATH' );`